tests/console/nftables.pm

```diff
@@ -0,0 +1,113 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: nftables
+# Summary: Test nftables with firewalld
+# Maintainer: Zaoliang Luo <zluo@suse.de>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # check nftables, stop iptables and start firewalld by need
+    my $logs = "/tmp/log";
+
+    #check that nftables has been already installed
+    assert_script_run('echo -e "____ nftables version and rules ____\n\n" > ' . $logs);
+    assert_script_run("rpm -qa nftables |& tee -a $logs");
+    assert_script_run("if systemctl is-active -q iptables; then systemctl stop iptables; fi");
+    assert_script_run("if ! systemctl is-active -q firewalld; then systemctl start firewalld; fi");
```

Comment:
Why the `if` - you could simply always stop and start that service.

Reply:
yes, just restart firewalld now:  http://10.160.64.152/tests/637#step/nftables/20